microDeposits.go

```diff
@@ -565,8 +565,11 @@ func (r *SQLDepositoryRepo) initiateMicroDeposits(id DepositoryID, userID string
 // or there are a mismatched amount the call will return a non-nil error.
 func (r *SQLDepositoryRepo) confirmMicroDeposits(id DepositoryID, userID string, guessAmounts []Amount) error {
```

Comment:
I did a bit of refactoring and tidying up in this function.  If you find it to be unnecessary, I can undo these changes.